src/adapters/conversant.js

```diff
@@ -0,0 +1,230 @@
+        'page': page
+      },
+
+      'device': getDevice(),
+      'at': 1
+    };
+
+    var url = secure ? 'https:' + conversantUrl : location.protocol + conversantUrl;
+    httpPOSTAsync(url, JSON.stringify(conversantBidReqs));
+  };
+
+  var addEmptyBidResponses = function (placementsWithBidsBack) {
+    var allConversantBidRequests = $$PREBID_GLOBAL$$._bidsRequested.find(bidSet => bidSet.bidderCode === 'conversant');
+
+    if (allConversantBidRequests && allConversantBidRequests.bids){
+      utils._each(allConversantBidRequests.bids, function (conversantBid) {
+        if (!utils.contains(placementsWithBidsBack, conversantBid.placementCode)) {
+          // Add a no-bid response for this placement.
+          var bid = bidfactory.createBid(2, conversantBid);
+          bid.bidderCode = 'conversant';
+          bidmanager.addBidResponse(conversantBid.placementCode, bid);
+        }
+      });
+    }
+  };
+
+  var parseSeatbid = function (bidResponse) {
+    var placementsWithBidsBack = [];
+    utils._each(bidResponse.bid, function (conversantBid) {
+      var responseCPM,
+        placementCode = '',
+        id = conversantBid.impid,
+        bid = {},
+        responseAd,
+        responseNurl,
+        sizeArrayLength;
+
+      // Bid request we sent Conversant
+      var bidRequested = $$PREBID_GLOBAL$$._bidsRequested.find(bidSet => bidSet.bidderCode === 'conversant').bids.find(bid => bid.bidId === id);
+
+      if (bidRequested) {
+        placementCode = bidRequested.placementCode;
+        placementsWithBidsBack.push(placementCode);
+        bidRequested.status = CONSTANTS.STATUS.GOOD;
+
+        responseCPM = parseFloat(conversantBid.price);
+
+        if (responseCPM !== 0.0) {
+          conversantBid.placementCode = placementCode;
+          conversantBid.size = bidRequested.sizes;
+          responseAd = conversantBid.adm || '';
+          responseNurl = conversantBid.nurl || '';
+
+          // Our bid!
+          bid = bidfactory.createBid(1);
+          bid.creative_id = conversantBid.id || '';
+          bid.bidderCode = 'conversant';
+
+          bid.cpm = responseCPM;
+
+          // Track impression image onto returned html
+          bid.ad = responseAd + '<img src=\"' + responseNurl + '\" />';
+
+          sizeArrayLength = bidRequested.sizes.length;
+          if (sizeArrayLength === 2 && typeof bidRequested.sizes[0] === 'number' && typeof bidRequested.sizes[1] === 'number') {
+            bid.width = bidRequested.sizes[0];
+            bid.height = bidRequested.sizes[1];
+          } else {
+            bid.width = bidRequested.sizes[0][0];
+            bid.height = bidRequested.sizes[0][1];
+          }
+
+          bidmanager.addBidResponse(placementCode, bid);
+        } else {
+          //0 price bid
+          //indicate that there is no bid for this placement
+          bid = bidfactory.createBid(2);
```

Comment:
pass the bid object as 2nd argument to sync bid request and response IDs. See #509